js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
+const pushButton = document.querySelector(".first-task button");
+const outArea = document.querySelector(".first-task .out-area");
```

Comment:
Please, also rename 'outArea' as I mentioned above and selector .first-task is redundant here

Reply:
done